storage/app/releases.dist.php

```diff
@@ -130,9 +168,19 @@
                 21, // PlayStation 2
             ],
         ],
+        Emulators::DuckStation => [
+            'name' => 'DuckStation',
+            'handle' => 'DuckStation',
+            'active' => true,
+            'link' => 'https://www.duckstation.org/wiki/Main_Page',
```

Comment:
This link is correct, but seems to be returning a hard 404 from DuckStation's web server.

Reply:
I noticed that too, but it's the link right off their page, and the only link available. The same thing happens if you take the `Main_Page` off - it can't find an index page either.

![image](https://user-images.githubusercontent.com/32680403/232258111-7d9d797d-c88d-42b7-91be-618a6f06ee45.png)

Reply:
Yeah, I double-checked this as well to be sure.

Taking a look at archive.org, it seems this link has been broken for at least over a month:
https://web.archive.org/web/20230315020146/https://www.duckstation.org/wiki/Main_Page

Feels like a not great experience for us to direct users to a 404 page.

Reply:
https://github.com/stenzek/duckstation/wiki There's the github wiki for it. At least it's something.